indexer/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@filecoin-station/spark-piece-indexer",
+  "type": "module",
+  "private": true,
+  "scripts": {
+    "dry-run": "node bin/dry-run.js",
+    "migrate": "node bin/migrate.js",
+    "start": "node bin/piece-indexer.js",
+    "lint": "standard",
+    "test": "node --test"
+  },
+  "devDependencies": {
+    "standard": "^17.1.0"
+  },
+  "dependencies": {
+    "debug": "^4.3.6",
+    "ioredis": "^5.4.1"
```

Comment:
What is `ioredis` used for here?

Reply:
Ah, it's not used here. I want to try using Redis instead of PG, as you did in spark-rewards, and I accidentally added ioredis to dependencies in the wrong PR.